### PR TITLE
#49: Added tutorial about how to start the VM/AMI

### LINKED ISSUE #49: Add tutorials about how to start the VM/AMI
## Background

We should provide a step-by-step tutorial for customers how to use the VM's for the following hypervisors:
- Virtual Box
- Boxes

and also how to start an EC2-instance based on the AMI:
- configure security group to get access to ssh and Jupyter
- launch EC-2 Instance

Also mention username (`ubuntu`) to the user guide.

### PR DESCRIPTION
**Pull request by tomuben**
fixes #49

### CODE REVIEW

File: doc/user_guide/user_guide.md [hunk 1]
```diff
@@ -43,7 +43,69 @@ We recommend the following requirements for the VM instance:
 | RAM  | 2GB     | 8GB         |
 | CPU  | 1 Core  | 2 Cores     |
 
-### Password
+### Use the AMI
+
+#### Overview
+
+1. Create a Security Group with open inbound ports for `ssh` (22) and `Jupyter` (8888)
```
Comment (tkilias) [inline doc/user_guide/user_guide.md:50]: are you sure, you want to expose jupyter to the world

Comment (tomuben) [reply doc/user_guide/user_guide.md:50]: Added a notice in the details about the security risk

File: doc/user_guide/user_guide.md [hunk 2]
```diff
@@ -43,7 +43,71 @@ We recommend the following requirements for the VM instance:
...
+1. Go to [AWS Console](https://aws.amazon.com/console/)
+2. Go to EC2 (in the search field at the top type EC2 and click on the result)
+3. Create a security group for ssh and (optionally) Jupyter:
+    - In the navigation bar on the left select "Security Groups"
+    - Click button "Create Security Group"
+    - Choose name and VPC
+    - For outbound rules keep the default
+    - Create  inbound rules: 
+      - One of type `ssh`
+      - If you plan to connect to the Jupyter lab, add another rule of type "Custom TCP" with port 8888. **Important**: With this rule you expose the Jupyter lab to the internet; anybody who has access to the password will be able to execute commands. For a minimum of security you should change the default Jupyter password. Details about how to do that will be shown in the login screen. But, you should consider to use only `ssh`.
```
Comment (tkilias) [inline doc/user_guide/user_guide.md]:
```suggestion
      - If you plan to connect to the Jupyter lab, add another rule of type "Custom TCP" with port 8888. 
        - **Important**: With this rule you expose the Jupyter lab to the internet; anybody who has access to the password will be able to execute commands. For a minimum of security you should change the default Jupyter password. Details about how to do that will be shown in the login screen. But, you should consider to use only `ssh` with port forwarding.
```

File: doc/user_guide/user_guide.md [hunk 3]
```diff
@@ -43,7 +43,71 @@ We recommend the following requirements for the VM instance:
...
+ 4. Go back to the EC2 console
+ 5. Launch the EC2 instance:
+     - In the navigation bar on the left select "Instances"
+     - Click button "Launch instances"
+     - At field "Application and OS Images" select the AMI id of the developer sandbox (found in the [release notes](https://github.com/exasol/script-languages-developer-sandbox/releases/latest))
+     - Select an appropriate instance type (at least "Small")
```
Comment (tkilias) [inline doc/user_guide/user_guide.md]: What is Small, this is not an instance type

File: doc/user_guide/user_guide.md [hunk 4]
```diff
@@ -43,7 +43,71 @@ We recommend the following requirements for the VM instance:
...
+     - Choose your key pair
+     - Choose the security group which your created in step 3.
+     - For the storage keep we recommend to keep the pre-selected 100GB volume
```
Comment (tkilias) [inline doc/user_guide/user_guide.md]:
```suggestion
     - For the storage we recommend to keep the pre-selected 100GB volume
```

**Review by tkilias — CHANGES_REQUESTED** [review]
see my comments

File: doc/user_guide/user_guide.md (hunk 3 above)
Comment (tomuben) [reply doc/user_guide/user_guide.md]: changed

File: doc/user_guide/user_guide.md [hunk 5]
```diff
@@ -43,7 +43,72 @@ We recommend the following requirements for the VM instance:
...
+2. Go to EC2 (in the search field at the top type EC2 and click on the result)
+3. Create a security group for ssh and (optionally) Jupyter:
+    - In the navigation bar on the left select "Security Groups"
+    - Click button "Create Security Group"
+    - Choose name and VPC
+    - For outbound rules keep the default
+    - Create  inbound rules: 
+      - One of type `ssh`
+      - If you plan to connect to the Jupyter lab, add another rule of type "Custom TCP" with port 8888. 
+        - **Important**: With this rule you expose the Jupyter lab to the internet; anybody who has access to the password will be able to execute commands. For a minimum of security you should change the default Jupyter password. Details about how to do that will be shown in the login screen. But, you should consider to use only `ssh` with port forwarding.
```
Comment (tkilias) [inline doc/user_guide/user_guide.md]:
```suggestion
        - **Important**: With this rule you expose the Jupyter lab to the internet; anybody who has access to the password will be able to execute commands. For a minimum of security you should change the default Jupyter password. Details about how to do that will be shown in the login screen. However, you should consider to use only `ssh` with port forwarding.
```

**Review by tkilias — APPROVED** [review]

**Pull request merged**